packaging/libbaresip.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
+exec_prefix=${prefix}
+libdir={$prefix}/lib
+includedir=${prefix}/include
+
+Name: libbaresip
+Description: @CMAKE_PROJECT_DESCRIPTION@
+Version: @PROJECT_VERSION@
+URL: @CMAKE_PROJECT_HOMEPAGE_URL@
+Libs: -L${libdir} -lbaresip
```

Comment:
```suggestion
Libs: -L${libdir} -lbaresip -lrem -lre
```

Reply:
Why? And if, shouldn't the `includedir` and/or `Cflags` be changed equivalent then, too? But shouldn't this go, if at all, into `Libs.private`, `Requires` or `Requires.private`?